src/nft/components/explore/Banner.tsx

```diff
@@ -36,152 +79,46 @@ const Banner = () => {
     }
   )
 
-  useEffect(() => {
-    /* Rotate through Top 5 Collections on 15 second interval */
-    let stale = false
-    if (hovered || stale) return
-    const interval = setInterval(async () => {
-      if (collections) {
-        const nextCollectionIndex = (current + 1) % collections.length
-        const nextCollectionAddress = collections[nextCollectionIndex].address
-        setCurrent(nextCollectionIndex)
-        await queryClient.prefetchQuery(['collectionActivity', nextCollectionAddress], () =>
-          ActivityFetcher(nextCollectionAddress as string)
-        )
-      }
-    }, 15_000)
-    return () => {
-      stale = true
-      clearInterval(interval)
-    }
-  }, [current, collections, hovered])
-
   return (
-    <Box onMouseEnter={() => setHover(true)} onMouseLeave={() => setHover(false)} cursor="pointer" width="full">
-      {collections && collections[current] ? (
-        <Link to={`/nfts/collection/${collections[current].address}`} style={{ textDecoration: 'none' }}>
-          <Box style={{ height: '386px' }}>
-            <div
-              className={styles.bannerWrap}
-              style={{ backgroundImage: `url(${collections[current].bannerImageUrl})` }}
-            >
-              <Box className={styles.bannerOverlay} width="full" />
-              <Box as="section" className={styles.section} display="flex" flexDirection="row" flexWrap="nowrap">
-                <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
-                {windowWidth && windowWidth > breakpoints.lg && <ActivityFeed address={collections[current].address} />}
-              </Box>
-
-              <CarouselProgress length={collections.length} currentIndex={current} setCurrent={setCurrent} />
-            </div>
-          </Box>
-        </Link>
+    <BannerContainer>
+      <HeaderContainer>
+        Best price. {!isMobile && <br />}
+        Every listing.
+      </HeaderContainer>
+      {collections ? (
```

Comment:
nit and non-blocking: but it would slightly reduce repeated code to re-order checks to be isMobile -> if collections has content. This would prevent having to declare the carousel twice